packages/hash/api/src/model/ontology/entity-type.model.ts

```diff
@@ -253,4 +256,14 @@ export default class {
 
     return undefined;
   }
+
+  isLinkEntityType(): boolean {
+    /**
+     * @todo: account for link entity types being able to inherit from other link entity types
```

Comment:
@hashintel/backend do we have an existing relevant task for this? Or should I create one?

Reply:
Probably this: https://app.asana.com/0/1200211978612931/1201726402115269/f

There's a graph specific one: https://app.asana.com/0/1201095311341924/1203277018227719/f

So could make another task for the TS backend to block the bigger one if you'd prefer.

Reply:
Thanks, added in https://github.com/hashintel/hash/pull/1435/commits/5017c624e28421bcfa46bf99ad0f6a915919008d